### PR TITLE
some enhancements & typos

### PR DESCRIPTION
**Pull request by acrepet**

### CODE REVIEW

File: client/src/app/component/production/production-content.directive.html [hunk 1]
```diff
@@ -26,8 +26,8 @@
             à appliquer des formats en utilisant des caractères spéciaux (* = * [] !).</label>
                     <textarea class="form-control" rows="10" ng-model="ctrl.production.content" id="content" name="content" ng-disabled="ctrl.isLocked"
                               ng-required="true"
-                              ng-maxlength="3000"></textarea>
-        <small ng-if="!frm.content.$error.maxlength">({{3000 - ctrl.production.content.length }}
+                              ng-maxlength="5000"></textarea>
```
Comment (javamind) [inline client/src/app/component/production/production-content.directive.html:27]: Is it not to high ? 
But If the users need this width, there's no problem, we use a clob in database. 
So if you want to merge let's go

Comment (acrepet) [inline client/src/app/component/production/production-content.directive.html:27]: ok for me!
OK for you the others points?

**Comment by acrepet** [comment]
if you are OK, you can validate and merge my PR

File: client/src/app/component/bigqrcode/bigqrcode.html [hunk 2]
```diff
@@ -85,8 +85,12 @@ <h1 uib-collapse="isCollapsed"><i class="glyphicon glyphicon-blackboard"></i> Mu
             </svg>
         </g>
     </svg>
+    <div class="row">
+        <br/>
```
Comment (javamind) [inline client/src/app/component/bigqrcode/bigqrcode.html]: bahhh it stinks :smile:

Comment (javamind) [inline client/src/app/component/bigqrcode/bigqrcode.html]: What do you want to do ?
If it's a break line use pure HTML  `<p>&nbsp;</p>` or the best styles

Comment (acrepet) [inline client/src/app/component/bigqrcode/bigqrcode.html]: I know ;-) but in fact the big QRCODE is stuck to the page footer, si I want to add a row space

File: client/src/app/component/production/production-content.directive.html [hunk 3]
```diff
@@ -21,27 +21,29 @@
...
-                    <textarea class="form-control" rows="10" ng-model="ctrl.production.content" id="content" name="content" ng-disabled="ctrl.isLocked"
+        <label for="content">Votre texte:</small>
+        <textarea class="form-control" rows="10" ng-model="ctrl.production.content" id="content" name="content" ng-disabled="ctrl.isLocked"
                               ng-required="true"
-                              ng-maxlength="3000"></textarea>
-        <small ng-if="!frm.content.$error.maxlength">({{3000 - ctrl.production.content.length }}
+                              ng-maxlength="5000"></textarea>
+        <small ng-if="!frm.content.$error.maxlength">({{5000 - ctrl.production.content.length }}
             caractère{{ctrl.production.content.length<=2999 ? 's' :
             ''}} restant{{ctrl.production.content.length<=2999 ? 's' : ''}})
```
Comment (javamind) [inline client/src/app/component/production/production-content.directive.html]: You need to update 2999 => 4999
I know my code stinks too... It's better to do a directive or a filter for that but time is money :smile:

File: client/src/app/component/bigqrcode/bigqrcode.html (hunk 2 above)
Comment (javamind) [inline client/src/app/component/bigqrcode/bigqrcode.html]: Ok

File: client/src/app/component/bigqrcode/bigqrcode.html [hunk 4]
```diff
@@ -85,8 +85,10 @@ <h1 uib-collapse="isCollapsed"><i class="glyphicon glyphicon-blackboard"></i> Mu
             </svg>
         </g>
     </svg>
+    <p>&nbsp;</p>
```
Comment (javamind) [inline client/src/app/component/bigqrcode/bigqrcode.html:88]: :+1:

**Pull request merged**